### PR TITLE
Describe of how to use the foreman_scc_manager to mange SUSE content

### PR DESCRIPTION
**Pull request by sbernhard**

### CODE REVIEW

File: plugins/katello/nightly/user_guide/suse_content/index.md [hunk 1]
```diff
@@ -6,21 +6,66 @@ version: nightly
 
 # SUSE Content
 
-Katello can be used to manage content associated with SUSE products based upon contents mirrored via the smt tool.  This includes content such as RPMs, errata.
+Katello can be used to manage content associated with SUSE products.  This includes content such as RPMs, errata.
+
+There are two different ways to use Katello to get manage SUSE products / repositories:
+1. Use foreman_scc_manager https://github.com/ATIX-AG/foreman_scc_manager
```
Comment (evgeni) [inline plugins/katello/nightly/user_guide/suse_content/index.md]: this probably should be [`foreman_scc_manager`]( https://github.com/ATIX-AG/foreman_scc_manager)? (a link, and in backticks for a gem name)

File: plugins/katello/nightly/user_guide/suse_content/index.md [hunk 2]
```diff
@@ -6,21 +6,66 @@ version: nightly
...
 - Repository - Collection of content (either rpm).
 - Product - Collection of repositories (content hosts attach to a product).
 - Library - The initial lifecycle environment where repositories are created.  Content that is synced or uploaded lands in the library.
 
-## General Workflow
+## Manage SUSE Content using the foreman_scc_manager
+
+To manage SUSE Content witht the foreman_scc_manager you need to have an SUSE SCC account
+
+### General Workflow
```
Comment (evgeni) [inline plugins/katello/nightly/user_guide/suse_content/index.md:24]: please add an empty line after this heading

File: plugins/katello/nightly/user_guide/suse_content/index.md [hunk 3]
```diff
@@ -6,21 +6,66 @@ version: nightly
...
+- Install the foreman_scc_manager (this will automatically restart foreman).
```
Comment (evgeni) [inline plugins/katello/nightly/user_guide/suse_content/index.md]: I'd drop the "will restart" part, that's a detail that will be described later.

File: plugins/katello/nightly/user_guide/suse_content/index.md [hunk 4]
```diff
@@ -6,21 +6,66 @@ version: nightly
...
+- Set up the connection to your SCC account.
+- Synchronize the list of available SUSE products.
+- Select the products and sub-products which you want to add to Katello (Content > Products). A ForemanTask adds then the products and repositories.
```
Comment (evgeni) [inline plugins/katello/nightly/user_guide/suse_content/index.md]: does the user need to know about the ForemanTask detail here?

File: plugins/katello/nightly/user_guide/suse_content/index.md [hunk 5]
```diff
@@ -6,21 +6,66 @@ version: nightly
...
+- After the previous step is done, you can synchronize the repositories.
```
Comment (evgeni) [inline plugins/katello/nightly/user_guide/suse_content/index.md]: I'd just write "Synchronize the repositories", no "after …", no "you can". To match the other bullet points

File: plugins/katello/nightly/user_guide/suse_content/index.md [hunk 6]
```diff
@@ -6,21 +6,66 @@ version: nightly
...
+
+### Installation
+
+The installation of the foreman_scc_manager can be started with the following command. This will restart your foreman!
```
Comment (evgeni) [inline plugins/katello/nightly/user_guide/suse_content/index.md]: `s/foreman/Foreman/` at the end

File: plugins/katello/nightly/user_guide/suse_content/index.md [hunk 7]
```diff
@@ -6,21 +6,66 @@ version: nightly
...
+
+![SCC page](/plugins/katello/{{ page.version }}/user_guide/suse_content/add_scc_account_window.png)
+
+The next step is to select the action "Sync" in the "Actions" drop down list. This will start the process to synchronize all available products for your SCC account. 
+
+![SCC page](/plugins/katello/{{ page.version }}/user_guide/suse_content/suse_subscriptions_page.png)
+
+
+Select the products which you want to include to Katello by the selecting the "Select products" in the "Actions" drop down list.
+This action will take some time as it will create a product and all repositories of each selected SUSE Product.
```
Comment (evgeni) [inline plugins/katello/nightly/user_guide/suse_content/index.md]: `s/Product/product/`

File: plugins/katello/nightly/user_guide/suse_content/index.md [hunk 8]
```diff
@@ -6,21 +6,66 @@ version: nightly
...
+
+![SCC page](/plugins/katello/{{ page.version }}/user_guide/suse_content/suse_subscriptions_page.png)
+
+
+Select the products which you want to include to Katello by the selecting the "Select products" in the "Actions" drop down list.
+This action will take some time as it will create a product and all repositories of each selected SUSE Product. 
+
+![SCC page](/plugins/katello/{{ page.version }}/user_guide/suse_content/scc_product_selection_window.png)
+
+After the ForemanTask (see Monitor > Tasks) is complete, you will see the list of selected products and its repositories in "Content > Products".
```
Comment (evgeni) [inline plugins/katello/nightly/user_guide/suse_content/index.md]: `s/ForemanTask/synchronization/`? the fact that it's a `ForemanTask` feels like an implementation detail the user does not necessarily need to know/understand?

**Comment by sbernhard** [comment]
Can you, @stbenjam / @ehelms provide some feedback regarding this PR, please?

**Review by stbenjam — APPROVED** [review]
I have no experience with the AITX plug-in but this all looks good to me.

**Comment by stbenjam** [comment]
Thanks @sbernhard !

**Comment by sbernhard** [comment]
Thanks for merging this @stbenjam

**Pull request merged**